Dockerfiles/primary_node/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM clusterdock/topology_nodebase:centos6.6
+
+# Build arguments.
+ARG MEP_VERSION
+ARG MAPR_VERSION
+
+ADD mapr_core.repo /tmp/mapr_core.repo
```

Comment:
Why not just put into the final location and then edit it in place? No need for a separate cp later.